control/bgp.yaml

```diff
@@ -116,4 +120,65 @@ components:
           format: uint32
           maximum: 3600
           default: 30
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
+        notification: 
+          description: |-
+            Send a Notification to the peer as per configured parameters when initially bringing down a session as per
+            configured parameters.
+          $ref: "#/components/schemas/Action.Protocol.BgpGracefulRestart.Notification"
+          x-field-uid: 3
+
+    Action.Protocol.BgpGracefulRestart.Notification:
+      description: >-
+        Defines the explicit contents of the NOTIFICATION message to be sent when executing InitiateGracefulRestart trigger. 
+        This causes the BGP connection to close.If a user wants to send custom Error Code and Error Subcode the custom object should be configured.
+        A user can send IANA defined BGP NOTIFICATIONs according to https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml. 
+      type: object
+      properties:       
+        choice:
+          description: >-
+            Each BGP NOTIFICATION message includes an Error Code field indicating what type of problem occurred. For certain Error Codes, an Error 
+            Subcode field provides additional details about the specific nature of the problem. 
+            The choice value will provide the Error Code used in NOTIFICATION message. 
+            The Subcode can be set for each of the corresponding errors except for Hold Timer Expired error and BGP Finite State Machine error. 
+            In both of these cases Subcode 0 will be sent. If a user wants to use non zero Sub Code then custom choice can be used.
+          type: string
+          x-field-uid: 2
+          default: cease
+          x-enum:
+            cease:
+              x-field-uid: 1
+            message_header_error:
+              x-field-uid: 2
+            open_message_error:
+              x-field-uid: 3
+            update_message_error:
+              x-field-uid: 4
+            hold_timer_expired:
+              x-field-uid: 5
+            finite_state_machine_error:
+              x-field-uid: 6
+            custom:
+              x-field-uid: 7
+
```

Comment:
same as previous comment related to space

Reply:
Want to keep space between diff. objects.